code/modules/camera_coverage/datum/controller/camera_coverage.dm

```diff
@@ -61,11 +69,15 @@ var/global/datum/controller/camera_coverage/camera_coverage_controller
 /datum/controller/camera_coverage/proc/update_turfs(list/turf/turfs_to_update)
 	if (!length(turfs_to_update))
 		return
+
+	LAZYLISTINIT(src.turf_update_queue)
 	for(var/turf/T as anything in turfs_to_update)
 		if (global.explosions.exploding || ON_COOLDOWN(T, "camera_coverage_update", CAM_TURF_UPDATE_COOLDOWN))
-			LAZYLISTADDUNIQUE(src.turf_update_queue, turfs_to_update)
+			src.turf_update_queue |= turfs_to_update
```

Comment:
why change this from lazy add unique

Reply:
doing an if for a fuckton of turfs tends to be slow